aodn/nrmn-application
# 4044 persist filters

filter restored even user enter page via menu, this is because localstorage value not rest properly

Test can be done via the following steps
1. Enter one of the page with filters (i.e. Site List)
2. Set a filter, click the button on the page to move to the next page.
3. Go back the previous page, filter persist
4. Go to a different page via left menu and then go back to Site List
5. Click the button on the page to move to the next page.
6. Go back to the previous page, the filter should not restore.
